### ISSUE TITLE
Avoiding affiliation with Minecraft

### ISSUE THREAD

**Issue by televisionia** [mustafakhafaji/mineblox#6]
I am incredibly worried about what could happen if this became a published project on Roblox, as I have seen many games rise and fall.

I have an idea where we could say it is inspired from the Minetest engine, which is also an open source, rather popular, voxel engine that is inspired by Minecraft.
You could also label it as a 'voxel game' instead of a Minecraft game.

**Comment by luckyboi1639** [comment]
In episode 1, he did say he changed the textures to stop copyright.

**Comment by yes133** [comment]
> In episode 1, he did say he changed the textures to stop copyright.

No, roblox games arent allowed to have blox in them anymore. Also, just changing the textures wont stop the copyright, he has to change the name.
